src/gui/transferlistwidget.cpp

```diff
@@ -931,6 +931,9 @@ void TransferListWidget::displayListMenu()
     TagSet tagsInAny;
     TagSet tagsInAll;
     bool hasInfohashV1 = false, hasInfohashV2 = false;
+    bool rechecking = false;
+    bool queued = false;
```

Comment:
I wouldn't add these two variables or at least move them into loop and declare as const.

Reply:
This will fix a bug with the current PR which is caused by the `else` parts when they are re-set to false: When selecting a queued and a paused torrent the setting is available. It only happens if the selected queued torrent is above the paused torrent in the list.

Reply:
Like I suggested above:
> ...move them into loop and declare as const.

Reply:
> > When selecting a **queued and a paused torrent** the setting is available.
> 
> Sounds incorrect. Paused and Queued are mutually exclusive states (in qBittorrent).

Hah what? You're gonna tell the user not to do that because these are "mutually exclusive states"?

Reply:
> > > When selecting a **queued and a paused torrent** the setting is available.
> > 
> > 
> > Sounds incorrect. Paused and Queued are mutually exclusive states (in qBittorrent).
> 
> Hah what? You're gonna tell the user not to do that because these are "mutually exclusive states"?

Apparently we didn't understand each other. I meant that a torrent cannot be both Paused and Queued at the same time.

Reply:
>Apparently we didn't understand each other. I meant that a torrent cannot be both Paused and Queued at the same time.

I was talking about 2 torrents.